README.template.md

```diff
@@ -70,6 +70,21 @@ echo 'export PATH="$PATH:$HOME/.bbin/bin"' >> ~/.zshrc && exec /bin/zsh
 echo 'export PATH="$PATH:$HOME/.bbin/bin"' >> ~/.bashrc && exec /bin/bash
 ```
 
+### Windows (Manual)
+
+**1. Install `bbin` CLI (and batch file, because Windows):**
+```shell
+> set BBIN_DEST=%HOMEDRIVE%%HOMEPATH%\.bbin\bin
+> mkdir %BBIN_DEST% && curl -o %BBIN_DEST%\bbin -L https://raw.githubusercontent.com/babashka/bbin/v{{version}}/bbin && curl -o %BBIN_DEST%\bbin.bat -L https://raw.githubusercontent.com/babashka/bbin/v{{version}}/bbin.bat
+> set BBIN_DEST=
+```
+
+**2. Add `%HOMEDRIVE%%HOMEPATH%\.bbin\bin` to `Path` environment variable**
+
+Because `Path` is often a system-level and user-level variable, setting from the command line can be messy; it's probably easiest to add it in the Environment Variables settings window.
```

Comment:
Not a blocker for this PR: If setting the `Path` through the GUI is the best way to do it, we should show some screenshots. When I was testing I had to Google to figure out where the "Environment Variables" settings window was.

With this in mind, I think we should include the extended instructions and screenshots in a separate `docs/windows.md` file and link to it from here so the main `README` doesn't get too big.